src/dbms/dbms_handler.hpp

```diff
@@ -48,6 +47,8 @@
 
 namespace memgraph::dbms {
 
+constexpr std::string_view kLSCTSKey = "last_commited_system_ts";
```

Comment:
Can stuff be moved to cpp? This implementation detail isn't needed to be public in other translation units.